.gitignore

```diff
@@ -121,3 +121,4 @@ build-cuberite
 # clang-tidy
 tidy-build
 run-clang-tidy.py
+/VS2019-x64
```

Comment:
You should put this into your local (per-machine) gitignore file - this is a folder name chosen by you, not by anything in the repo, and so is unrelated to the code here.